### PR TITLE
Workaround for issue where ProcDump64 swallows return codes.

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Resolves: #467 

Signed-off-by: Alan Jowett <alanjo@microsoft.com>

### CODE REVIEW

**Review by dthaler — APPROVED** [review]
Looks ok but would be good to add a `#` comment in the file explaining the tests are run twice.

**Review by dthaler — APPROVED** [review]
Looks ok, as long as you verified it with both a successful and unsuccessful test oass

**Comment by Alan-Jowett** [comment]
> Looks ok, as long as you verified it with both a successful and unsuccessful test pass

Unit test cashes:
https://github.com/microsoft/ebpf-for-windows/actions/runs/1187080997

Failed, but not crashing test:
https://github.com/microsoft/ebpf-for-windows/actions/runs/1187126702

And this run show the success case.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -82,15 +87,15 @@ jobs:
...
+      run: ./unit_tests.exe -s
 
     - name: Run RPC Client Tests
       working-directory: ./${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}
-      run: procdump64.exe -accepteula -o -ma -e -x c:/dumps/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}} ./ebpf_client.exe -s
+      run: ./ebpf_client.exe -s
 
-    - name: Run Performance Benchmark Tests
-      working-directory: ./${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}
-      run: procdump64.exe -accepteula -o -ma -e -x c:/dumps/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}} ./ebpf_performance.exe
```
Comment (saxena-anurag) [inline .github/workflows/build.yml:93]: We will not be running ebpf_performance.exe in the CI/CD?

Comment (Alan-Jowett) [reply .github/workflows/build.yml:93]: Good catch. Was removed by accident.

Comment (Alan-Jowett) [reply .github/workflows/build.yml:93]: Put back.

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**